pr_agent/tools/pr_code_suggestions.py

```diff
@@ -168,14 +168,15 @@ async def run(self):
                 get_logger().info('Code suggestions generated for PR, but not published since publish_output is False.')
         except Exception as e:
             get_logger().error(f"Failed to generate code suggestions for PR, error: {e}")
-            if self.progress_response:
-                self.progress_response.delete()
-            else:
-                try:
-                    self.git_provider.remove_initial_comment()
-                    self.git_provider.publish_comment(f"Failed to generate code suggestions for PR")
-                except Exception as e:
-                    pass
+            if get_settings().config.publish_output:
```

Comment:
move this to line 176

Reply:
shouldn't the progress_response not exist if publish_output is false anyway?

Reply:
i dont know. ok